src/apps/relay/mainrelay.c

```diff
@@ -1501,40 +1501,65 @@ static void set_option(int c, char *value)
 		add_static_user_account(value);
 		break;
 	case 'b':
+        {
 #if defined(TURN_NO_SQLITE)
-	  TURN_LOG_FUNC(TURN_LOG_LEVEL_WARNING, "WARNING: Options -b, --userdb and --db are not supported because SQLite is not supported in this build.\n");
+          TURN_LOG_FUNC(TURN_LOG_LEVEL_WARNING, "WARNING: Options -b, --userdb and --db are not supported because SQLite is not supported in this build.\n");
 #else
-		STRCPY(turn_params.default_users_db.persistent_users_db.userdb, value);
-		turn_params.default_users_db.userdb_type = TURN_USERDB_TYPE_SQLITE;
+            STRCPY(turn_params.default_users_db.persistent_users_db.userdb, value);
+            turn_params.default_users_db.userdb_type = TURN_USERDB_TYPE_SQLITE;
+            char *s_userdb = sanitize_userdb_string(turn_params.default_users_db.persistent_users_db.userdb);
```

Comment:
maybe this new code could go outside of the switch/case to avoid duplicating it 5 times.  Something like:

if (strlen(turn_params.default_users_db.persistent_users_db.userdb)) {
            char *s_userdb = sanitize_userdb_string(turn_params.default_users_db.persistent_users_db.userdb);
            STRCPY(turn_params.default_users_db.persistent_users_db.s_userdb, s_userdb);
            free(s_userdb);
}